### DISCUSSION TITLE
I9305 build?

### DISCUSSION THREAD

**Discussion by K4sum1** [Mibo5354/i9300_builds#8] [General]
They seem to have the same hardware aside form the I9305 having LTE and depending on the model an extra GB of memory. Is there any big reason to not make a I9305 build?

**Comment by Mibo5354** [comment]
I don't have a i9305, someone else can build it.

**Comment by K4sum1** [comment]
Now I'm dumb, say I get a I9305 or want to try making a build with the newer compiler and whatever. How exactly would I go about trying that?

**Comment by Mibo5354** [comment]
Easiest way is to use the https://github.com/lineageos4microg/docker-lineage-cicd docker image.
The build command I used is on the README of this repo. Just replace i9300 with i9305.
Also use my local manifests, https://github.com/Mibo5354/i9300_builds/tree/master/local_manifests except for [roomservice.xml](https://github.com/Mibo5354/i9300_builds/blob/master/local_manifests/roomservice.xml)
The official lineageOS repo doesn't have all the security patches merged.

Alternatively there are also [these instructions](https://wiki.lineageos.org/devices/i9305/build) from LineageOS.

**Comment by K4sum1** [comment]
Ok, I think I want to try building this, but I don't currently have a S3. Should I try just going for it, or try to build for a different phone that I already have?

I've never successfully built a ROM before, and good instructions/sources of information that actually work are very hard to find.
